code/src/index.css

```diff
@@ -5,9 +5,24 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: teal;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+#root {
```

Comment:
I understand we should try to avoid styling with ID, so maybe it's better to add a class to this root div and style using that class' name instead?